index.html

```diff
@@ -5,11 +5,239 @@
+
+                    </div>
+
+                </div>
+                <div class="card-4-speaker">
+                    <div class="s-portrait">
+                        <img class="f-photo" src="images/Maribel.png" alt="meribel">
+                        <img clas="b-portrait" id="b-portrait" src="images/portrait.png" alt="portrait">
+                    </div>
+                    <div class="details">
+                        <h3 class="detail-h-3">Lawrence Grace</h3>
+                        <h4 class="detail-h-4">USA</h4>
+                        <div class="detail-line"></div>
+                        <p class="detail-p-1">We are here to showcase our technology in military business and a better life for all</p>
+
+                    </div>
+
+                </div>
+                <div class="card-4-speaker">
+                    <div class="s-portrait">
+                        <img class="f-photo" src="images/Maribel.png" alt="meribel">
+                        <img clas="b-portrait" id="b-portrait" src="images/portrait.png" alt="portrait">
+                    </div>
+                    <div class="details">
+                        <h3 class="detail-h-3">Lawrence Grace</h3>
+                        <h4 class="detail-h-4">USA</h4>
+                        <div class="detail-line"></div>
+                        <p class="detail-p-1">We are here to showcase our technology in military business and a better life for all</p>
+
+                    </div>
+
+                </div> -->
+                <!-- <button class="btn-mobile-only more" type="button">More</button> -->
+
+            </div>
+            <div class="speaker-grid" id="speaker-all"></div>
+            <button class="btn-mobile-only show-more" id="show-more" type="button">Show more</button>
+            <button class="btn-mobile-only show-less" id="show-less" type="button">Show less</button>
+
+        </section>
+        <section class="partner">
+            <div class="h-partner">
+                <h2 class="h-1-partner">Partners</h2>
+            </div>
+
+            <div class="grid flex-1">
+                <div class="img-partner">
+                    <img src="images/Ferrero.png" alt="ferrero">
+                </div>
+                <div class="img-partner">
+                    <img src="images/Lindt.png" alt="Lindt">
+                </div>
+
+
+                <div class="img-partner">
+                    <img src="images/R_cote.png" alt="cote">
+                </div>
+                <div class="img-partner">
+                    <img src="images/Valhorna.png" alt="Valhorna">
+                </div>
+
+            </div>
+
+        </section>
+        <footer>
+            <div class="contact">
+                <p class="contact-a">Dubai Expo 2021</p>
+                <p class="contact-b">Dubai expo <br> Some right reserved </p>
+
+            </div>
+        </footer>
```

Comment:
- [ ] Let's address the following in the footer section.

| **The provided design** |
| --- |
| ![footer-desktop-design](https://user-images.githubusercontent.com/45002654/133837759-f0316bb7-a5e5-4148-901d-9aa1d558f087.PNG) |

| **Your Implementation** |
| --- |
| ![footer-imp](https://user-images.githubusercontent.com/45002654/133837800-dd3dbd8b-afd0-4bd5-84d9-5d7cd113f749.PNG) |

- It should be dark in color for the Desktop view.
- It should be white in color for the Mobile view.
- The logo and the text in the footer are separated by just some margin, in the implementation, they are separated by a very big margin. Work on this, please.

Reply:
- [ ] This change has not been addressed yet. Please try and implement all the changes mentioned out by the reviewer.